website_sale_wishlist/tests/test_ui.py

```diff
@@ -1,15 +1,15 @@
 # -*- coding: utf-8 -*-
 # Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
 # License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
-from openerp.tests.common import HttpCase
+from odoo.tests.common import HttpCase
 
 
 class UICase(HttpCase):
     def test_ui_website(self):
         """Test frontend tour."""
         self.phantom_js(
             url_path="/",
-            code="odoo.__DEBUG__.services['web.Tour']"
+            code="odoo.__DEBUG__.services['web_tour.tour']"
                  ".run('test_website_sale_wishlist', 'test')",
```

Comment:
A little tip: if you just copy this code and paste it in your browser's ES console, you'll see the tour running in front of your eyes. useful for debugging 😊